Source/WebKit/UIProcess/API/Cocoa/_WKWebExtensionContext.h

```diff
@@ -294,6 +294,14 @@ WK_CLASS_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA))
  */
 @property (nonatomic, readonly) BOOL hasAccessToAllHosts;
 
+/*!
+ @abstract Checks if the extension has script or stylesheet content that can be injected into the specified URL.
+ @param url The webpage URL to check.
+ @result Returns `YES` if the extension has content that can be injected by matching the `url` against the extension's requested match patterns.
+ @discussion The extension context will still need to be loaded and have granted website permissions for its content to actually be injected.
```

Comment:
`The extension context will still need to be loaded and have granted website permissions for its content to actually be injected.`

It would be nice if this caveat was made a bit more clear in the function name. Maybe `specifiesInjectedContentForURL`? I don't feel strongly about this though - what you have is fine.

Reply:
Yeah `specifiesInjectedContentForURL:` might be a good name for this. I'll go with that and see what API review says.